README.md

```diff
@@ -1 +1,162 @@
 # ApiTestCase
+[![Build Status](https://travis-ci.org/Lakion/ApiTestCase.svg?branch=master)](https://travis-ci.org/Lakion/ApiTestCase)
+
+**ApiTestCase** is an one-file library that will make your life as a Symfony2 API developer much easier. It extends basic [Symfony2](https://symfony.com/) WebTestCase with some cool features. Thanks to [*PHP-Matcher*](https://github.com/coduo/php-matcher) you can write expected json responses like a gangster(According to php-matcher README.md). [*SymfonyMockerContainer*](https://github.com/PolishSymfonyCommunity/SymfonyMockerContainer) makes it super easy to mock services. This library merges these two great concepts and extends them with a few more lines, to achieve pure awesomeness.
```

Comment:
I'd add that the matcher allows you for great TDD of your API. Because this is the main purpose of this library, do TDD of JSON APIs and this is how we should "advertise" it, wdyt?